config/vufind/searches.ini

```diff
@@ -634,13 +634,16 @@ view=full
 ; geographic points are located. If your dataset is not concentrated in one 
 ; geographic area, it is advised that you pick a default area, and do not use
 ; the entire extent of the map for searching (otherwise the search may be slow).
-; The default coordinates specified below are in decimal degrees, and are 
-; ordered as WENS (west, east, north, south). Ranges of valid values are:
-; -180 to 180 (longitude) and -85 to 85 (latitude). Note, to search from and to
-; the international date line, use west = -179 and east = -180. 
-default_coordinates = "-95, 30, 72, 15"
-; height: Height in pixels of the map selection interface.
-height = 320
+; Options:
+; default_coordinates:  The default coordinates specified below are in decimal
+;            degrees, and are ordered as WENS (west, east, north, south). Ranges of
+;            valid values are:; -180 to 180 (longitude) and -85 to 85 (latitude).
+;            Note, to search from and to the international date line, use
+;            west = -179 and east = -180.
+;
+; height:    Height in pixels of the map selection interface.
+default_coordinates = "-168, -50, 83, 25"
```

Comment:
The default coordinates have changed here. Is this intentional? If so, we can probably merge the reformatting and adjustment of this .ini file independently of the other unrelated changes....

Reply:
I will change that back to what it was. Thanks for catching that.  Those are the default coordinates I was using on my development server.

Reply:
Thanks!